packages/clay-dropdown/src/ClayDropdownBase.soy

```diff
@@ -10,21 +10,27 @@
 	{@param? caption: string}
 	{@param? expanded: bool}
 	{@param? helpText: string}
+	{@param? id: string}
 	{@param? indicatorsPosition: string}
 	{@param? searchable: bool}
 	{@param? spritemap: string}
 	{@param? style: string}
 	{@param? triggerClasses: string}
 	{@param? type: string}
 
-	{let $wrapperClasses kind="text"}
-		dropdown
-		{if $expanded}
-			{sp}show
+	{let $attributes kind="attributes"}
+		classes="dropdown
```

Comment:
This should not be `class="`?

Reply:
Sure, I corrected it later when I realized that 😅